msi/rust.wxs

```diff
@@ -9,12 +9,14 @@
         <Package
             Comments="Rust is a systems programming language that runs blazingly fast, prevents almost all crashes, and eliminates data races."
             InstallerVersion="200"
+            InstallPrivileges="limited"
```

Comment:
Without this UAC request is shown for both permachine and peruser installs.
With this UAC request is never shown, so to install permachine user shoud explicitly run installer as admin.